src/database/operations/load/loadUserBundle.ts

```diff
@@ -0,0 +1,43 @@
+import { loadBundle } from './loadBundle';
+import type {
+	JsonResultSet,
+	Bundle,
+	Proposal,
+	AuthContext,
+} from '../../../types';
+
+export const loadUserBundle = async (
+	queryParameters: {
+		offset: number;
+		limit: number;
+		authenticationId?: string;
+	},
+	authContext?: AuthContext,
+): Promise<Bundle<Proposal>> => {
+	const defaultQueryParameters = {
+		authenticationId: '',
+		userId: 0,
+		isAdministrator: false,
+	};
+	const { offset, limit, authenticationId } = queryParameters;
+	const userId = authContext?.user.id;
+	const isAdministrator = authContext?.role.isAdministrator;
+
+	const bundle = await loadBundle<JsonResultSet<Proposal>>(
```

Comment:
Same question here: `Proposal` or `User`?

Reply:
Fixed up!  Thank you.